core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -877,7 +877,7 @@ public Void call() {
             }
 
             try {
-                getChannelOrFail().setProperty("slave",Boolean.TRUE); // indicate that this side of the channel is the slave side.
+                getChannelOrFail().setProperty("slave",Boolean.TRUE); // indicate that this side of the channel is the agent side.
             } catch (ChannelClosedException e) {
                 throw new IllegalStateException(e);
             }
```

Comment:
A double usage "agent agent" at line 106.

Reply:
There are a few other usages of "slave" that could easily be cleaned up. Line 549 for example. Not as important but would still be good to clean up.